.github/workflows/release.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         with:
           dry_run: true
           extra_plugins: |
-            git+https://github.com/pbrisbin/semantic-release-stack-upload.git#pb/prepare-in-verify
+            git+https://github.com/pbrisbin/semantic-release-stack-upload.git
```

Comment:
should it be pinned to the SHA of what is currently `main` or do we want to just always take it as "latest" or whatever

Reply:
I chose convenience over safety :shrug:

I _just_ wrote this plugin, hence the git-reference. I do plan to release to npm soon.